lib/source-map.js

```diff
@@ -462,6 +462,8 @@ class SourceMap {
 
         if (sourceMap.mapCommentType === 'line') {
           stream.write('//# sourceMappingURL=' + sourceMappingUrl + '\n');
+        } else if (sourceMap.mapCommentType === 'hidden') {
+          // sourceMaps are hidden
         } else {
           stream.write('/*# sourceMappingURL=' + sourceMappingUrl + ' */\n');
```

Comment:
Suggestion:

```js
        if (sourceMap.mapCommentType === 'line') {
          stream.write('//# sourceMappingURL=' + sourceMappingUrl + '\n');
        } else if (sourceMap.mapCommentType !== 'hidden') {
          stream.write('/*# sourceMappingURL=' + sourceMappingUrl + ' */\n');
        }
```

Or to make things more clear:

```js
        switch (sourceMap.mapCommentType) {
          case 'line':
            stream.write('//# sourceMappingURL=' + sourceMappingUrl + '\n');
            break;
          case 'hidden':
            break;
          default:
            stream.write('/*# sourceMappingURL=' + sourceMappingUrl + ' */\n');
            break;
        }   
```

I think the switch is the most clear approach.